Sources/BraintreeCore/BTHTTP.swift

```diff
@@ -365,15 +320,6 @@ class BTHTTP: NSObject, NSCopying, URLSessionDelegate {
             return
         }
 
-        // We should only cache the response if we do not have an error and status code is 2xx
-        let successStatusCode: Bool = httpResponse.statusCode >= 200 && httpResponse.statusCode < 300
```

Comment:
Do we need to have this logic live elsewhere or is there no chance we could cache a config with a bad status code?

Reply:
We don't anymore, since now we're caching at the BTAPIClient level, and not the BTHTTP level. We only add a config to the cache if we can [successfully construct a Config object](https://github.com/braintree/braintree_ios/blob/95eb1b379bc568ee8a7a86f5369351b9e63c5320/Sources/BraintreeCore/BTAPIClient.swift#L149-L152), after the HTTP call completes